dockerfiles/nginx.conf

```diff
@@ -13,10 +13,41 @@ server {
   root /usr/src/app/public;
   index index.html;
 
+  # gzip
+  gzip on;
+  gzip_vary on;
+  gzip_proxied any;
+  gzip_comp_level 6;
+  gzip_types text/plain text/css text/xml application/json application/javascript application/rss+xml application/atom+xml image/svg+xml;
```

Comment:
Wondering if basic hhtp library handles gzipped `application/json` data

Reply:
`gunzip: on` should take care of this. Adding it.
https://docs.nginx.com/nginx/admin-guide/web-server/compression/#enabling-decompression

Reply:
Did you try this config with Python's `requests` for example?

Reply:
Yes, it works as expected. The config is up on alpha if you want to check.